### PR TITLE
Changes to README.md

### LINKED ISSUE #4: Commands (and very likely permissions) are broken on Gradle 4.10
When I try to create a new command, then Gradle throws an exception and mentions something about "Command with name 'creategame' not found.". Here's the stacktrace: https://paste.wut.ee/avecocenoz.cs  
I guess that something changed in Gradle API? :thinking:  
It works in 4.8 perfectly. I tried both 0.2.1 and 0.3.0 plugin versions.

I have this in my `build.gradle.kts`:

```kotlin
bukkit {
    name = "x"
    description = "y"
    authors = listOf("mikroskeem")
    main = "z"

    commands {
        "creategame" {
            description = "Creates a new game"
        }
    }
}
```

Current workaround is to use this:
```kotlin
bukkit {
    // ...
    commands.create("creategame") {
        description = "Creates a new game"
    }
}
```

### PR DESCRIPTION
**Pull request by Proximyst**
https://github.com/Minecrell/plugin-yml/commit/1c14564a2ccfd088a03ed9eae505e4ea58ba9658 closes #4, and https://github.com/Minecrell/plugin-yml/commit/67ff2fe3d4e7b39faa888f35fa93863d038ed471 simply fixes the Groovy example for Nukkit.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -212,7 +212,7 @@ nukkit {
     prefix = 'TEST'
     
     commands {
-        test {
+        'test' {
```
Comment (stephan-gh) [inline README.md]: Hmm, what's the problem with this? The same is suggested for Bukkit so apparently it has worked at some point :)

Comment (Proximyst) [reply README.md]: I was under the impression you had to use strings here, unless Gradle/Groovy does some magic I'm unaware of?

Comment (stephan-gh) [reply README.md]: Well, Groovy contains loooooooooooooooooooooooooooots of magic :D

Comment (Proximyst) [reply README.md]: Would you like me to just revert that change, in that case?

Comment (stephan-gh) [reply README.md]: I was hoping you could confirm that this indeed fixes something that is broken :)

Comment (Proximyst) [reply README.md]: Seems it doesn't from a test. Groovy sure is funky... I'll revert that.

**Review by stephan-gh — APPROVED** [review]

**Comment by stephan-gh** [comment]
Had to merge this manually because of conflicts and since you already deleted your fork. But I understand, this is mostly my fault for not merging this earlier :D Thanks!

**Pull request closed without merging**